Source/core/Time.h

```diff
@@ -26,7 +26,44 @@
 namespace WPEFramework {
 namespace Core {
     class EXTERNAL Time {
-    private:
+
+    public:
+        class TimeAsLocal {
+        private:
+            friend class Time;
+
+            TimeAsLocal(Time time)
+                : _time(time)
+            {
+            }
+
+        public:
+            TimeAsLocal(const TimeAsLocal& copy)
+                : _time(copy._time)
+            {
+            }
+	    inline TimeAsLocal& operator=(const TimeAsLocal& RHS)
+            {
+                _time = RHS._time;
+
+                return (*this);
+            }
+            TimeAsLocal() = delete;
+            ~TimeAsLocal() = default;
+
+            const Time& LocalTime() const
+            {
+                return _time;
+            }
+            Time& LocalTime()
+            {
+                return _time;
+            }
+
+        private:
+            Time& _time;
```

Comment:
Don't think this will work correctly. It is now a reference to a temporary. Better to just make it a non reference member

Reply:
changed no reference